### PR TITLE
NC-1

### PR DESCRIPTION
**Pull request by axpreparator**

### CODE REVIEW

File: blocklearning/lib/app/data/provider/storage_provider.dart [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import 'package:get_storage/get_storage.dart';
+
+class StorageProvider extends GetxService {
+  final box = GetStorage();
+
+  Future init() async {
+    await GetStorage.init();
+  }
+
+  void addSmsSend() {
```
Comment (dadsparta) [inline blocklearning/lib/app/data/provider/storage_provider.dart:13]: Where will we using it?

Comment (dadsparta) [inline blocklearning/lib/app/data/provider/storage_provider.dart:13]: Where will we using it?

File: blocklearning/lib/app/data/provider/storage_provider_keys.dart [hunk 2]
```diff
@@ -0,0 +1,4 @@
+abstract class StorageProviderKeys {
```
Comment (dadsparta) [inline blocklearning/lib/app/data/provider/storage_provider_keys.dart:1]: Why we use some smsDateKey? We won't use it!

**Review by dadsparta — COMMENTED** [review]
Please, answare to my questions.

File: blocklearning/lib/app/data/provider/storage_provider.dart (hunk 1 above)
Comment (axpreparator) [reply blocklearning/lib/app/data/provider/storage_provider.dart:13]: No. I lost it for example

File: blocklearning/lib/app/data/provider/storage_provider_keys.dart (hunk 2 above)
Comment (axpreparator) [reply blocklearning/lib/app/data/provider/storage_provider_keys.dart:1]: It's for exaple using providers and getx storage

**Review by dadsparta — APPROVED** [review]

**Review by starikandrei — APPROVED** [review]

**Pull request merged**